src/main/g8/.scalafmt.conf

```diff
@@ -0,0 +1,22 @@
+style = defaultWithAlign
```

Comment:
I'd suggest either removing this scalafmt file or providing a basic sbt configuration to make this file valid